src/frontend/utils/video-helpers.js

```diff
@@ -0,0 +1,7 @@
+export const getVideoUrl = id => {
+	return `https://youtu.be/${id}`;
```

Comment:
No need to use return here, You can use it like this => **export const getVideoUrl = id => `https://youtu.be/${id}`**

Reply:
Yes, will make the changes